microprofile/microprofile-openapi-1.0/src/main/java/org/wildfly/swarm/ts/microprofile/openapi/v10/MyResponse.java

```diff
@@ -1,9 +1,7 @@
 package org.wildfly.swarm.ts.microprofile.openapi.v10;
 
-import javax.xml.bind.annotation.XmlRootElement;
 import java.io.Serializable;
 
-@XmlRootElement
```

Comment:
On Java 11, JAXB isn't present in the JDK, so I'd have to add a Maven dependency, which would be a waste of time, because it's not really needed. We don't serialize anything to XML here.